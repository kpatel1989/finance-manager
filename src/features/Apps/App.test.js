import React from 'react';
import renderer from 'react-test-renderer';
import App from './index';

jest.mock('../Home', () => () => (<homeplaceholder />));

test('renders learn react link', () => {
  const component = renderer.create(<App />, document.body);
  expect(component.toJSON()).toMatchSnapshot();
});
