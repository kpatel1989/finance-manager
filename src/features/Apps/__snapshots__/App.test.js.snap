// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <span
      className="title"
    >
       Your financial Database 
    </span>
  </header>
  <homeplaceholder />
</div>
`;
